Assets/MixedRealityToolkit/EventDatum/Input/HandTrackingInputEventData.cs

```diff
@@ -30,7 +30,7 @@ public class HandTrackingInputEventData : InputEventData<Vector3>
         /// <param name="touchedObject">This is a the global position of the HandTrackingInputSource that created the EventData</param>
         public void Initialize(IMixedRealityInputSource inputSource, IMixedRealityController controller, Handedness sourceHandedness, Vector3 touchPoint)
```

Comment:
>sourceHandedness [](start = 116, length = 16)

If this is intentionally ignored, it might be worth leaving a comment saying so (i.e. it's weird that we're being told a source handedness for hand tracking data and then saying it doesn't matter)

Reply:
Hmm yeah, does seem weird that it's ignored. Not sure that was intentional. (Also, looks like the method summary has some stale parameters)

Reply:
Whoops I didn't refresh!